docs/visual-basic/language-reference/modifiers/private-protected.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Private Protected (Visual Basic)"
+ms.date: 05/10/2018
+helpviewer_keywords: 
+  - "Private Protected keyword [Visual Basic]"
+  - "Private Protected keyword [Visual Basic], syntax"
+---
+# Private Protected (Visual Basic)
+
+The `Private Protected` keyword combination is a member access modifier. A `Private Protected` member is accessible by all members in its containing class, as well as by types derived from the containing class, but only if they are found in its containing assembly. 
+
+You can specify `Protected Friend` only on members of classes; you cannot apply `Protected Friend` to members of a structure because structures cannot be inherited.
+
+The `Private Protected` access modifier is supported starting with Visual Basic 15.5. To use the `Private Protected` access modifier, you must add the following element to your Visual Basic project (*.vbproj) file:
+
+```xml
```

Comment:
I'd add "15.5 or newer", or the "latest" value for the LangVersion element. That future proofs these changes.